core/src/main/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulation.kt

```diff
@@ -271,6 +276,12 @@ fun distributeAllowance(
     startOffset: Offset<TravelledPath>,
     endOffset: Offset<TravelledPath>
 ): List<AllowanceRange> {
+    assert(startOffset <= endOffset)
+    if (startOffset == endOffset) {
+        // TODO: raise warning (overlapping scheduled points)
+        standaloneSimLogger.info("Warning: different scheduled points at the same location")
```

Comment:
Nit (feel free to resolve and merge): For monitoring, this log is more important than the one on [ScheduleItem parsing](https://github.com/OpenRailAssociation/osrd/pull/7747/files?diff=split&w=1#diff-1d19928711ba9b2048694a68df42df6685708db9697b0fb8e753ae54d3ed3e05R52).

So I would `log.warn` here and downgrade to `log.info` there (or `log.error` here)